syft/util.py

```diff
@@ -0,0 +1,78 @@
+from syft import check
+import inspect
+
+
+@check.type_hints
+def is_static_method(klass: type, attr: str):
+    """Test if a value of a class is static method.
+
+    Example:
+
+        class MyClass(object):
+            @staticmethod
+            def method():
+                ...
+
+    Args:
+        klass (type): the class on which we want to check whether the method is statically implemented
+        attr (str): the name of the method we want to check.
+
+    Returns:
+        bool: whether or not a method named <attr> is a static method of class <klass>
+    """
+
+    if not inspect.isclass(klass):
+        return False
+
+    if hasattr(klass, attr):
+        value = getattr(klass, attr)
+    else:
+        return False
+
+    assert getattr(klass, attr) == value
```

Comment:
What is this `assert` doing exactly? Checking that the class method is idempotent? Can it be written without `assert`?